ProcessLib/ChemistryLib/PhreeqcInterfaceData/EquilibriumPhase.h

```diff
@@ -0,0 +1,53 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <boost/optional/optional.hpp>
+#include <fstream>
+#include <string>
+#include <vector>
+
+namespace BaseLib
+{
+class ConfigTree;
+}
+
+namespace ProcessLib
+{
+namespace ChemistryLib
+{
+class EquilibriumPhase
+{
+public:
+    EquilibriumPhase(std::string name, double amount, double saturation_index)
+        : _name(name), _amount(amount), _saturation_index(saturation_index)
+    {
+    }
+
+    std::string const& getName() const { return _name; }
+
+    double const& getAmount() const { return _amount; }
+
+    void setAmount(double amount) { _amount = amount; }
```

Comment:
make `_amount` public, drop the set/get methods. Same for name, make it public and keep constness, drop the getter.

Reply:
It is not possible to remove the setter and make it public. There are multiple places in PhreeqcInterface.cpp where the setter function is called to update the amount of equilibrium phase (see line 68, 248, 263, 276 in PhreeqcInterface.cpp). I have removed the getter since it is not used externally.

Reply:
Removed get-/set- member functions from all the classes which contain them (class Component, class EquilibriumPhase, and class KineticReactant). The basic class type of Component is changed to struct.

Reply:
Solution still has a get/set PH.

Reply:
Removed.